### PR TITLE
Bug 2015115: [RFE] PCI passthrough - align hardware list rows

### PR DESCRIPTION
**Pull request by avivtur**
Align hardware rows to fit into wizard

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/HardwareDevicesList/HardwareDevicesList.tsx [hunk 1]
```diff
@@ -8,18 +8,21 @@ import {
...
+
 export type HardwareDevice = {
   // Common denominator of V1GPU and V1HostDevice
   deviceName: string; // 'deviceName' is the resource name of the host device exposed by a device plugin
   name: string; // Name of the host device as exposed by a device plugin
 };
 
 export type HardwareDevicesListProps = {
-  devices: HardwareDevice[];
+  devices: HardwareDevice[] | any[];
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/HardwareDevicesList/HardwareDevicesList.tsx:20]: why ?
this doesn't look right ?

File: frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx [hunk 2]
```diff
@@ -163,6 +163,7 @@ export const HardwareDevicesModal = withHandlePromise(
                     : t('kubevirt-plugin~No Host devices found')
                 }
                 isUserForbidden={isUserForbidden}
+                className="pf-c-empty-state"
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx:166]: why ?

File: frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx [hunk 3]
```diff
@@ -163,6 +163,7 @@ export const HardwareDevicesModal = withHandlePromise(
                     : t('kubevirt-plugin~No Host devices found')
                 }
                 isUserForbidden={isUserForbidden}
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx:165]:
```suggestion
                disabled={isUserForbidden}
                // or
                // editable={!isUserForbidden}
```
if a user is forbidden we "disable"  input fields (e.g. delete and add) of show the editable version of the list

File: frontend/packages/kubevirt-plugin/src/components/HardwareDevicesList/HardwareDevicesList.tsx (hunk 1 above)
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/src/components/HardwareDevicesList/HardwareDevicesList.tsx:20]: +1

File: frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx (hunk 2 above)
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx:166]: we want the empty state will have a centralized text only when it's a modal, so this is a prop that should be inherited from the modal component IMO

File: frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx (hunk 3 above)
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx:165]: +1

**Pull request merged**